stable/airflow/templates/_helpers.tpl

```diff
@@ -70,39 +90,23 @@ Create a random string if the supplied key does not exist
 {{- end -}}
 
 {{/*
-Create the name for the airflow secret.
+Create a set of environment variables to be mounted in web, scheduler, and woker pods.
+For the database passwords, we actually use the secretes created by the postgres and redis sub-charts.
 */}}
-{{- define "airflow.secret" -}}
-    {{- if .Values.existingAirflowSecret -}}
-        {{- .Values.existingAirflowSecret -}}
-    {{- else -}}
-        {{ template "airflow.fullname" . }}
-    {{- end -}}
-{{- end -}}
-
-{{/*
-Map environment vars to secrets
-*/}}
-{{- define "airflow.mapenvsecrets" -}}
-    {{- $secretName := printf "%s-env" (include "airflow.fullname" .) }}
-    {{- $mapping := .Values.airflow.defaultSecretsMapping }}
-    {{- if .Values.existingAirflowSecret }}
-      {{- $secretName := .Values.existingAirflowSecret }}
-      {{- if .Values.airflow.secretsMapping }}
-        {{- $mapping := .Values.airflow.secretsMapping }}
-      {{- end }}
-    {{- end }}
-    {{- range $val := $mapping }}
-      {{- if $val }}
-  - name: {{ $val.envVar }}
+{{- define "airflow.mapenvsecrets" }}
```

Comment:
This new template is greatly simplified and does not have the scoping error in the previous version.